tests/phpstan/bootstrap.php

```diff
@@ -1,17 +1,20 @@
 <?php
 
 $rootDir = getenv('_PS_ROOT_DIR_');
-
 if (!$rootDir) {
     echo '[ERROR] Define _PS_ROOT_DIR_ with the path to PrestaShop folder' . PHP_EOL;
     exit(1);
 }
 
+$pathToModuleRoot = __DIR__ . '/../../';
+
 // Add module composer autoloader
-require_once dirname(__DIR__) . '/../vendor/autoload.php';
+require_once $pathToModuleRoot . 'vendor/autoload.php';
```

Comment:
```suggestion
 require_once __DIR__ . '/../../vendor/autoload.php'; 
```

Reply:
I aggree with the suggested changes but they can't be applied (the diff is incorrect).

Reply:
Fixed with two new suggestions ;)